features/admin.applications.v1/components/edit-application.tsx

```diff
@@ -222,724 +231,164 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
     };
 
     /**
-     * Resolves the tab panes based on the application config.
-     *
-     * @returns Resolved tab panes.
+     * Todo Remove this mapping and fix the backend.
```

Comment:
Is there an issue to track this?

Reply:
This is an existing code block. https://github.com/wso2/identity-apps/blob/05605b9367987c95ce06add63ce0e95494a90b2d/features/admin.applications.v1/components/edit-application.tsx#L784-L786